presto-main/src/main/java/com/facebook/presto/server/json/JsonCodecConfig.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.server.json;
+
+import io.airlift.configuration.Config;
+
+public class JsonCodecConfig
+{
+    private boolean includeAfterBurnerModule;
+
+    @Config("json.module.after-burner.enabled")
```

Comment:
We should find a better name for this config, how about `experimental.json-serde-codegen-enabled`?

Then we should also rename the field/method in this class accordingly.

Reply:
Once we do that I think we can move this config property to `FeaturesConfig` class.